src/sentry/workflow_engine/models/action.py

```diff
@@ -24,17 +24,13 @@ class Type(models.TextChoices):
         Notification = "SendNotificationAction"
         TriggerWorkflow = "TriggerWorkflowAction"
 
-    """
-    Required actions cannot be disabled by the user, and will not be displayed in the UI.
-    These actions will be used internally, to trigger other aspects of the system.
-    For example, creating a new issue in the Issue Platform or a detector emitting an event.
-    """
-    required = models.BooleanField(default=False)
-
     # The type field is used to denote the type of action we want to trigger
     type = models.TextField(choices=Type.choices)
     data = models.JSONField(default=dict)
 
+    # TODO - finish removing this field
```

Comment:
JFYI: I've got an operation coming soon that will make deleting these things simpler https://github.com/getsentry/sentry/pull/81098